cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "projectId": "slate",
+  "env": {
+    "SITE_URL": "localhost:3000"
```

Comment:
Instead of defining a `$SITE_URL` and a `goTo` command that works on that, perhaps it would be possible to set Cypress' `baseUrl` via an environment variable and just use `cy.visit('/example/app')` - just prepend `CYPRESS_BASE_URL=https://localhost:3000` to the cypress app command in `package.json` or elsewhere, or add it to where environment variables are defined.  See https://docs.cypress.io/guides/guides/environment-variables#Setting